site/docs/v1/tech/core-concepts/integration-points.adoc

```diff
@@ -86,6 +86,37 @@ Additionally, when you use the hosted login pages, FusionAuth provides transpare
 
 The alternative to using the hosted login pages is building your own login experience. You can then use the APIs or an OAuth grant to authenticate your user against FusionAuth. This alternative gives you more control at the cost of more development effort.
 
+== SAML Integration Options
+
+You can use SAML to integrate with FusionAuth in a few different ways, depending on the role that FusionAuth is playing and your other needs.
+
+.SAML Integration options
+[cols="1,1,1,1"]
+|===
+| FusionAuth is the | Who initiates SSO? | Replay protection? | More info
+
+|SP
+|SP
+|Yes, using `InResponseTo` element
+|link:/docs/v1/tech/identity-providers/samlv2/[SAML Identity Provider]
+
+|SP
+|IdP, sends unsolicited AuthN response to SP
+|Yes, using internal mechanism
+|Coming soon: SAML IdP Initiated Identity Provider documentation
+
+|IdP
+|SP
+|N/A
```

Comment:
This is fine, or we could spell out that it is up to the SP to prevent replay attacks using our signed response.